packages/app-desktop/gui/MasterPasswordDialog/PasswordStrengthMeter.js

```
@@ -0,0 +1,36 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.default = PasswordStrengthMeter;
+const React = require("react");
+function PasswordStrengthMeter(props) {
+    const { score, label, suggestions, ruleIssues, Iscompromised } = props;
```

ESLint

[error] Expected indentation of 1 tab but found 4 spaces.

(indent)